device-types/Juniper/PTX10001-36MR.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Juniper
+model: PTX10001-36MR
+slug: ptx10001-36mr
+interfaces:
+  - name: re0:mgmt-0
+    type: 1000base-t
+    mgmt_only: true
+  - name: et-0/0/0
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/1
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/2
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/3
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/4
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/5
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/6
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/7
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/8
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/9
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/10
+    type: 100gbase-x-qsfp28
+  - name: et-0/0/11
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/0
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/2
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/3
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/4
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/5
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/6
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/7
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/8
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/9
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/10
+    type: 100gbase-x-qsfp28
+  - name: et-0/1/11
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/0
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/1
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/2
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/3
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/4
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/5
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/6
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/7
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/8
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/9
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/10
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/11
+    type: 100gbase-x-qsfp28
+power-ports:
+  - name: PSM0
+    type: iec-60320-c20
+    maximum_draw: 3000
+  - name: PSM1
+    type: iec-60320-c20
+    maximum_draw: 3000
```

Comment:
It appears this devices PSUs are modular. Thus, you should have these defined under `module-bays` and the applicable PSU modules should be added to the module-types folder if they do not already exist
It also appears this device has 6 modular fans. Those should be added to `module-bays` as well

Reply:
I can remove the power-ports information. Not a problem.
Again however, a lot of juniper device do not report that level of detail.

Reply:
if you look at MX104-BASE or MX104-PREMIUM for example, you'll see they report PEM modules as built-in power-ports.